app/src/test/java/uk/ac/ebi/atlas/experimentpage/markergenes/JsonMarkerGenesControllerWIT.java

```diff
@@ -78,12 +79,13 @@ void setUp() {
         this.mockMvc = MockMvcBuilders.webAppContextSetup(this.wac).build();
     }
 
-    @Test
+    @RepeatedTest(10)
```

Comment:
Changed this from @Test to @RepeatedTest - I know this test is a flaky one that fails very randomly or any other reason for this test to run 10 times?